index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Mobile First Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <style> @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap'); </style>
+  <style> @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap'); </style>
+  <style> @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap'); </style>
+</head>
+<body>
+  <header>
+    <div class="toolbar">
+      <a href="https://github.com/bilalrajput09" class="name">
+        Bilal Ahmed
+      </a>
+        <img src="C:\Users\HP\My-Portfolio\img\threelines.png">
+    </div>  
```

Comment:
- [ ] Good job at the project, but I see that the image of the hamburger not appearing ,so kindly try to add the correct image 👌

| Figma design | Your design| 
| --- | --- |
|   ![image](https://user-images.githubusercontent.com/83360636/212333179-46b5e2fd-7639-495f-bf77-edd0b93ed7cf.png)|    ![image](https://user-images.githubusercontent.com/83360636/212333109-a418e078-cca4-440e-a0c4-c4cf12d845fc.png) |